sean/joke_style.css

```diff
@@ -0,0 +1,79 @@
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+  }
+  
+  .content {
+    font-family: 'Rubik', sans-serif;
+    color: #f5f4f1;
+    justify-content: center;
+    align-items: center;
+    margin: 100px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    background-image: linear-gradient(45deg, #000000 0%, #1a181d 100%);
+  }
+  
+  .header {
+   text-align: center;
+   margin-top: 20px;
+  }
+  
+  .joke {
+    text-align: center;
+    margin-top: 35px;
+  }
+  
+  .img__wrap {
+    position: relative;
+    /* display: block; */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    top: 75px;
+    height: 477px;
+    width: 365px;
+  }
+  
+  .text_overlay{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* look up display flex 
+    justify and align-items: ;*/
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(3, 3, 3, 0.72);
+    visibility: hidden;
```

Comment:
Let's go ahead and remove all instances of `visibility:hidden` and `visibility:visible` from the CSS. `opacity` is already doing all that work for us, _and_ it is accessible. Visibility hidden will __also__ hide the element from a screen reader, so they wouldn't be able to get the joke.

The reason it works right now is because you bring back the visibility property of `.pline` on line 77.